assets/css/style.css

```diff
@@ -0,0 +1,234 @@
+body {
+    background-color: #d9dcd6;
+    background-image: url('../images/grapes.jpg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    min-height: 100vh;
+}
+
+
+main {
+    display: flex;
+    justify-content: center;
+}
+
+header{
```

Comment:
Issue: Fix space between `header` and `{`. 

Idea: You should use Prettier for formatting in all of your code. It's a super easy way to make sure most formatting issues are covered, and it's pretty much industry standard now. 

I like to use the "Format on Save" option in VSCode: https://stackoverflow.com/a/54665086/16763751

Extenstion: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
Why Prettier: https://prettier.io/docs/en/why-prettier.html